engine/src/main/java/io/camunda/zeebe/process/test/engine/db/Bytes.java

```diff
@@ -56,9 +57,10 @@ public int compareTo(final Bytes other) {
       final byte ourByte = byteArray[i];
       final byte otherByte = otherByteArray[i];
 
-      if (ourByte < otherByte) {
+      final int compared = UnsignedBytes.compare(ourByte, otherByte);
```

Comment:
🔧 Instead of depending on the external library, we can convert values to `int` using Java's `Byte.toUnsignedInt(byte x);` and keep the original comparison. 👍

Reply:
There's also `Arrays#compareUnsigned` which does exactly what we want and probably faster than what we can implement.

Reply:
Thank you for the suggestion, sounds great 👍🏻